### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by nissim229aws**
Please review

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1 +1,7 @@
-readme
+# Nissim Readme formats 1#
+## Nissim Readme formats 2#
+### Nissim Readme formats 3#
+#### Nissim Readme formats 4#
+* Nissim Readme formats 1*
+: Nissim Readme formats 1:
+🧮
```
Comment (nissim229) [inline readme.md:7]: try more formats

Comment (nissim229aws) [reply readme.md:7]: No need for mre

Comment (nissim229) [reply readme.md:7]: ok

Comment (nissim229) [reply readme.md:7]: add license file

Comment (nissim229aws) [reply readme.md:7]: License added

Comment (nissim229) [reply readme.md:7]: Thanks

**Review by nissim229 — APPROVED** [review]

**Pull request merged**